generated/nifgen/_library.py

```diff
@@ -44,6 +44,7 @@ def __init__(self, ctypes_library):
         self.niFgen_DefineUserStandardWaveform_cfunc = None
         self.niFgen_DeleteNamedWaveform_cfunc = None
         self.niFgen_DeleteScript_cfunc = None
+        self.niFgen_DeleteWaveformDispatch_cfunc = None
```

Comment:
These shouldn't be in the _library.py.

Reply:
Removed. Also removed WriteWaveformDispatcher since it shouldn't be there either.